confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConf.java

```diff
@@ -18,18 +18,21 @@
  */
 package org.apache.iotdb.confignode.conf;
 
+import org.apache.iotdb.commons.conf.IoTDBConstant;
```

Comment:
Constants between DataNode and ConfigNode might different. Is that a good idea for blend them with a single IoTDBConstant class?

Reply:
We put the same constants into IoTDBConstant， Different constants are stored separately